### DISCUSSION TITLE
What does the dimensional information of the frame represent?

### DISCUSSION THREAD

**Discussion by BugMaker2002** [Dylan-H-Wang/SLF-RPM#9] [Q&A] [answered]
Hello, when I run this code, the output align_frames.shape is (150, 7, 64, 64, 3), I want to know what 7 stands for
![image](https://github.com/Dylan-H-Wang/SLF-RPM/assets/110211332/c98ac454-806b-4c91-8763-2df0813fe342)

**Comment by Dylan-H-Wang** [comment, marked as answer]
If you read the paper and check the original code at 
https://github.com/Dylan-H-Wang/SLF-RPM/blob/298b01fc47d06506b71eda220e4e231e58f83a66/utils/utils.py#L102
and 
https://github.com/Dylan-H-Wang/SLF-RPM/blob/298b01fc47d06506b71eda220e4e231e58f83a66/utils/utils.py#L424
you will know it represents 7 ROIs.

**Reply by BugMaker2002** [reply]
thank you very much
